ksql-engine/src/main/java/io/confluent/ksql/analyzer/ExpressionAnalyzer.java

```diff
@@ -125,6 +126,15 @@ public Object visitColumnReference(
       return null;
     }
 
+    @Override
+    public Object visitDereferenceExpression(
```

Comment:
The only job of the `ExpressionAnalyzer` is to find ambiguous or missing `ColumnNameRef`s.  So all we need to do here is process the base of the `DereferenceExpression` as it may contain `ColumnNameRef`s.